### PR TITLE
Issue 7266 Handling same and different admin port for flush to storage command

### LINKED ISSUE #7266: Retrieve ADMIN CLI ports configured from Env Variables while flushting to storage.
**Describe the bug**
Retrieve ADMIN CLI ports configured, from Env Variables while flushting to storage.
Not doing so with the latest operator results in calling FlushToStorage with ports incremented over the configured port which is not required in environments like OpenShift.

**To Reproduce**
Run FlushToStorage on Openshift with the latest operator to see it reproduced.

**Screenshots**
N/A

**Additional information**
n/a

### PR DESCRIPTION
**Pull request by kuldeepk3**
Signed-off-by: Kuldeep Kumar <kuldeep.kumar3@dell.com>

**Change log description**  
Handling different or same port issue while executing slushToStorage command on different set ups .

**Purpose of the change**  
Fixes #7266 

**What the code does**  
Handling different or same port issue while executing slushToStorage command on different set ups .

**How to verify it**  
Execution of flush to storage command should be successful

### CODE REVIEW

**Review by abhinb — APPROVED** [review]

File: cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/FlushToStorageCommand.java [hunk 1]
```diff
@@ -46,6 +47,7 @@ public class FlushToStorageCommand extends ContainerCommand {
 
     private static final int REQUEST_TIMEOUT_SECONDS = 60 * 30;
     private static final String ALL_CONTAINERS = "all";
+    private boolean useAdminPort = true;
```
Comment (sachin-j-joshi) [inline cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/FlushToStorageCommand.java:50]: rename it to something more descriptive like `useFixedAdminPort`

**Review by Amit-Singh40 — COMMENTED** [review]
LGTM.

**Review by Amit-Singh40 — APPROVED** [review]

File: cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/FlushToStorageCommand.java (hunk 1 above)
Comment (kuldeepk3) [reply cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/FlushToStorageCommand.java]: updated

**Review by sachin-j-joshi — APPROVED** [review]
LGTM

**Review by abhinb — APPROVED** [review]

**Comment by kuldeepk3** [comment]
> Waiting for confirmation of passing tests

Tests are executed and we have required code coverage.

**Pull request merged**